File: pset1/mario1.c
#include <stdio.h>
#include "cs50.h"

int main(void)
{
    int h = 0;
    do
    {
        h = get_int("Height: ");
    }
    while (h < 1 || h > 8);
    for (int i = 0; i < h; ++i)
    {
        for (int j = 0; j < h; ++j)
        {
            if (i + j < h - 1)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("\n");
    }
    return 0;
}

File: pset6/mario1/mario1.py
from cs50 import get_int


def main():
    h = get_positive_int()
    for i in range(1, h + 1):
        for j in range(1, h + 1):
            if i + j <= h:
                print(" ", end="")
            else:
                print("#", end="")
        print()


def get_positive_int():
    while True:
        n = get_int("Height: ")
        if n > 0 and n < 9:
            break
    return n


main()
